sumomongodbatlascollector/api.py

```diff
@@ -214,22 +214,20 @@ def transform_data(self, content):
                 msg['cluster_name'] = cluster_name
                 current_date = msg['ts']['$date']
             else:
-                msg = {
-                    'msg': line,
```

Comment:
can we check whether it's possible to create version 3 mongodb engine? if not then we can remove this code, if version 3 is still supported then we have to make this code backward compatible.

Reply:
Mongo version 3 is not supported in clusters. We can remove this code then.

Reply:
Which part of code exactly?